lib/rubocop/cop/style/indent_heredoc.rb

```diff
@@ -17,20 +17,29 @@ module Style
       #   END
       #
       #   # good
+      #   # When EnforcedStyle is ruby23, bad code is auto-corrected to the
+      #   # following code.
       #   <<~END
       #     something
       #   END
       #
       #   # good
-      #   <<~END.strip_heredoc
+      #   # When EnforcedStyle is active_support, bad code is auto-corrected to
+      #   # the following code.
+      #   <<-END.strip_heredoc
       #     something
       #   END
       class IndentHeredoc < Cop
         include ConfigurableEnforcedStyle
 
-        MSG = 'Use %d spaces for indentation in a heredoc, ' \
-              'relative to the start of the line where the ' \
-              'heredoc delimiter is.'.freeze
+        # TODO: <<-
+        RUBY23_MSG = 'Use %d spaces for indentation in a heredoc by using ' \
+                     '`<<~` instead of `%s`.'.freeze
+        LIBRARY_MSG = 'Use %d spaces for indentation in a heredoc by using ' \
+                      '`String#strip_heredoc` that is provided by ' \
+                      'ActiveSupport.'.freeze
+        HOW_TO_CONFIGURE_MSG = 'You need to configure EnforcedStyle to use ' \
```

Comment:
I find this message pretty confusing. It's it natural that auto-correction is related to the enforced style?